CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## [v1.3.0](https://github.com/skyway/skyway-js-sdk/releases/tag/v1.3.0) - 2019-05-21
+
+### Added
+
+- Add an option when call `Connection.close` to signal to the remote peer to be disconnected instantly.
```

Comment:
How about `Add an option when calling `Connection.close` to signal intention to disconnection to the remote peer instantly.` ?

Reply:
sounds quite clearly to me. I'll modify it.